base/inference.jl

```diff
@@ -3276,44 +3276,6 @@ function replace_getfield!(ast, e::ANY, tupname, vals, sv, i0)
     end
 end
 
-function code_typed(f, types::ANY; optimize=true)
-    types = to_tuple_type(types)
-    code_typed(call, Tuple{isa(f,Type)?Type{f}:typeof(f), types.parameters...}, optimize=optimize)
```

Comment:
no, and that's the current behavior. This `call` refers to `Core.Inference.call` because of the file that it is in, not `Main.Base.call`.